src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/EFLWindowInfo.cs

```diff
@@ -0,0 +1,49 @@
+﻿using ElmSharp;
+using System;
+using Tizen.Applications.ComponentBased.Common;
+
+namespace Tizen.Applications.ComponentBased.Default
+{
+    /// <summary>
+    /// Window information class for ComponentBasedApplication
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class EFLWindowInfo : IWindowInfo
+    {
+        private const string LogTag = "Tizen.Applications.EFLWindow";
+        private EvasObject _win;
+        private int _resId;
+
+        /// <summary>
+        /// Initializes the EFLWindow class.
+        /// </summary>
+        /// <param name="win">The window object of component.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public EFLWindowInfo(EvasObject win)
```

Comment:
Why not use `Window` type? `EvasObject` could not be `Window`
Are you don't care is it not window?
```suggestion
        public EFLWindowInfo(Window win)
```

Reply:
Changed it into Window